lib/jekyll_pages_api/page.rb

```diff
@@ -59,11 +59,13 @@ def skip_index?
     def to_json
       optional = {}
       optional['skip_index'] = true if self.skip_index?
+
       optional.merge({
         title: self.title,
         url: self.url,
         tags: self.tags,
-        body: self.body_text
+        body: self.body_text,
+        meta: self.page.data
```

Comment:
Mind adding this field to the example in the README?

https://github.com/18F/jekyll_pages_api#usage